src/helpers/regex.helper.test.ts

```diff
@@ -0,0 +1,33 @@
+import { regexToSting } from './regex.helper';
+
+describe('Regex Helper', () => {
+  describe('regexToSting', () => {
+    it('replaces the leading and trailing forward slashes with an empty string', () => {
+      const regex = /test/;
+      const regexString = regexToSting(regex);
+
+      expect(regexString).toBe('test');
+    });
+
+    it('escapes regexp forward slashes inside a string', () => {
+      const regex = /test\/test/;
+      const regexString = regexToSting(regex);
+
+      expect(regexString).toBe('test\\/test');
```

Comment:
I'm a little tempted in these tests to convert the string back to regexp and compare (`new RegExp(string)`). Happy to stick or change, depending on what reviewers think

Reply:
I've updated to include both these assertions